core/logger/zap.go

```diff
@@ -86,31 +90,30 @@ func newCores(cfg ZapLoggerConfig) ([]zapcore.Core, error) {
 		err = diskErr
 	}
 
-	for _, sink := range cfg.sinks {
-		cores = append(
-			cores,
-			zapcore.NewCore(
-				zapcore.NewJSONEncoder(makeEncoderConfig(cfg.local)),
-				sink,
-				zap.LevelEnablerFunc(cfg.Level.Enabled),
-			),
-		)
-	}
-
 	return cores, err
 }
 
-func newConsoleCore(cfg ZapLoggerConfig) zapcore.Core {
-	filteredLogLevels := zap.LevelEnablerFunc(cfg.Level.Enabled)
-
+func newConsoleCore(cfg ZapLoggerConfig) (zapcore.Core, zapcore.WriteSyncer, error) {
```

Comment:
this is no longer just console - could be anything in paths, so maybe rename to just newCore ?